caikit/runtime/http_server/http_server.py

```diff
@@ -141,6 +141,21 @@ def __init__(self, tls_config_override: Optional[aconfig.Config] = None):
 
         self.app = FastAPI()
 
+        # Request validation
+        @self.app.exception_handler(RequestValidationError)
+        async def request_validation_exception_handler(_, exc: RequestValidationError):
+            err_code = status.HTTP_422_UNPROCESSABLE_ENTITY
+            error_content = {
+                "details": exc.errors()[0]["msg"]
+                if len(exc.errors()) > 0 and "msg" in exc.errors()[0]
+                else exc.errors(),
+                "additional_info": exc.errors(),
+                "code": err_code,
+                "id": None,
```

Comment:
Not sure if we want to include an always null ID though? 🤔

Reply:
What ids are normally set here by other services that follow this convention? Are they internal ids similar to log codes?

Reply:
Raising a `CaikitRuntimeException` adds a `uuid.uuid4().hex` to the err.id, which we then use. Would it make sense to add it here too?

```
            except CaikitRuntimeException as err:
                error_code = GRPC_CODE_TO_HTTP.get(err.status_code, 500)
                error_content = {
                    "details": err.message,
                    "code": error_code,
                    "id": err.id,
                }
```

Reply:
Right, I have no context on if those IDs are supposed to be
1. unique per instance of an error, in order to help you trace a single failure through a system or
2. unique per error case, so that you could identify the root cause in the code, like we use log codes for

I was wondering what other services that follow the `id / details / code` do. I'd be fine going with case (1) for now though since I dunno how we'd handle (2) effectively across all the places our code is scattered.

> Would it make sense to add it here too?

TL;DR: yeah I think so. Let's make sure that ID is logged by our server before returning the response so that somebody who is digging through logs can find that log statement, and use it to grab a thread id or something that they can use to find more details down the stack

Reply:
Makes sense!